apps/storybook/src/stories/transfer-button/transfer-button.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Input } from "@/components/ui/input";
+import { Meta, StoryObj } from "@storybook/react";
+import { useState } from "react";
+import { TransferButton } from "./transfer-button";
+import { withMockAccount, withWagmiProvider } from "../decorators/wagmi";
+import { Label } from "@/components/ui/label";
+import { Account } from "viem";
+interface TransferButtonProps {
+  to: string;
+  account?: Account;
+}
+
+const TransferButtonStory = ({ to, account }: TransferButtonProps) => {
+  const [amount, setAmount] = useState(0.001);
+
+  return (
+    <div className="flex flex-col gap-2">
+      <Label>Amount</Label>
+      <div className="flex gap-2 relative">
+        <Input type="number" value={amount} onChange={(e) => setAmount(Number(e.target.value))} />
+        <p className="absolute right-3 top-0 bottom-0 flex items-center">ETH</p>
+      </div>
```

Comment:
nit: think we should avoid overlapping.

<img width="217" alt="image" src="https://github.com/user-attachments/assets/0aca8eb1-36c1-46bf-97c8-3916fdd25350">